tslib/src/config.ts

```diff
@@ -34,7 +35,7 @@ export default {
     database: process.env.SQL_DATABASE ?? 'mtc',
     connectionTimeout: parseInt(parser.valueOrSubstitute(process.env.SQL_CONNECTION_TIMEOUT, oneMinuteInMilliseconds), 10),
     censusRequestTimeout: parseInt(parser.valueOrSubstitute(process.env.SQL_CENSUS_REQUEST_TIMEOUT, twoHoursInMilliseconds), 10),
-    requestTimeout: parseInt(parser.valueOrSubstitute(process.env.SQL_REQUEST_TIMEOUT, oneMinuteInMilliseconds), 10),
+    requestTimeout: parseInt(parser.valueOrSubstitute(process.env.SQL_REQUEST_TIMEOUT, tenMinutesInMilliseconds), 10),
```

Comment:
putting aside the actual reason for this - other requests using the same config that are performing common db updates could tie up connections for a long time in a bottleneck scenario.  Best practice suggests short timeouts so connections are not tied up for long.

It's like 2 different use cases, and i wonder if we should have a separate setting for 'long running' data operations, that you can utilise for the sync service instead.  Maybe we should discuss further.